modules/src/ics07_tendermint/client_state.rs

```diff
@@ -0,0 +1,82 @@
+use crate::ics02_client::client_type::ClientType;
+use crate::ics23_commitment::CommitmentRoot;
+
+use crate::ics07_tendermint::header::Header;
+use crate::ics24_host::client::ClientId;
+use serde_derive::{Deserialize, Serialize};
+use std::time::Duration;
+use tendermint::lite::Header as liteHeader;
+
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub struct ClientState {
+    id: String,
+    trusting_period: Duration,
+    unbonding_period: Duration,
+    latest_header: Header,
+    frozen_height: crate::Height,
```

Comment:
The order in the go-code is different (latest header should be last):

```suggestion
    frozen_height: crate::Height,
    latest_header: Header,
```

Reply:
Oy! good catch! thx, just changed. The problem is that we cannot test properly with SDK since we don't deserialize yet. Once we move to protobuf and we do the decoding we will catch more of these.